src/data/ozw.ts

```diff
@@ -7,15 +13,80 @@ export interface OZWDevice {
   is_failed: boolean;
   is_zwave_plus: boolean;
   ozw_instance: number;
+  event: string;
 }
 
+export interface OZWDeviceMetaDataResponse {
+  node_id: number;
+  ozw_instance: number;
+  metadata: OZWDeviceMetaData;
+}
+
+export interface OZWDeviceMetaData {
+  OZWInfoURL: string;
+  ZWAProductURL: string;
+  ProductPic: string;
+  Description: string;
+  ProductManualURL: string;
+  ProductPageURL: string;
+  InclusionHelp: string;
+  ExclusionHelp: string;
+  ResetHelp: string;
+  WakeupHelp: string;
+  ProductSupportURL: string;
+  Frequency: string;
+  Name: string;
+  ProductPicBase64: string;
+}
+
+export const getIdentifiersFromDevice = function (
+  device: DeviceRegistryEntry
+): OZWNodeIdentifiers | false {
+  if (typeof device === "undefined") return false;
```

Comment:
```suggestion
  if (!device) {
    return false;
  }
```

Reply:
We should not return false here, but we should return undefined.

Als the caller could check if the device is undefined before calling this function.